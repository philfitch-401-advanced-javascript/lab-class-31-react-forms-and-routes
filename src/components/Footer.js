import React from 'react';

const Footer = () => (
  <p>Â© 2019</p>
);

export default Footer;
