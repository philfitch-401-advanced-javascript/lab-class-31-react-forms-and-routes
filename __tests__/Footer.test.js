import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../src/components/Header';

describe('Footer component', () => {
  it('renders Footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
