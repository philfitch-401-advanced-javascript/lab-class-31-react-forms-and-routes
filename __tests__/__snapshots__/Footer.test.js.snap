// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders Footer 1`] = `
<nav>
  <Link
    to="/"
  />
  <Link
    to="/"
  />
  <Link
    to="/"
  />
</nav>
`;
